src/Jackalope/Transport/Jackrabbit/Client.php

```diff
@@ -1245,7 +1245,14 @@ private function storeProperty(Property $property)
                 $this->setJsopBody('^' . $path . ' : ');
             }
         } else {
-            $this->setJsopBody('^' . $path . ' : ' . json_encode($value));
+            $encoded = json_encode($value);
+
+            if (PropertyType::DOUBLE == $property->getType()
+                && !strpos($encoded, '.')
+            ) {
+                $encoded .= '.0';
+            }
+            $this->setJsopBody('^' . $path . ' : ' . $encoded);
```

Comment:
`setJsopBody` also accepts 3 parameters with the 3rd being the type. but using that does not work out, we run into other errors.